types/relay-runtime/index.d.ts

```diff
@@ -62,8 +62,6 @@ export {
     FragmentReference,
     FragmentSpecResolver,
     HandleFieldPayload,
-    LogEvent,
```

Comment:
This types actually exists on `relay-runtime` source code.

Check this https://github.com/facebook/relay/blob/master/packages/relay-runtime/index.js#L119-L120

Reply:
not in 6.0.0: https://github.com/facebook/relay/blob/b7501c6472b34ae2b0bb8cc164a648bb90749d74/packages/relay-runtime/index.js#L126-L127

Reply:
Fair enough. I was looking at master branch.